hal/halx86/xbox.cmake

```diff
@@ -35,16 +36,14 @@ list(APPEND HAL_XBOX_SOURCE
     xbox/part_xbox.c
     xbox/halinit_xbox.c
     xbox/reboot.c
+    up/irql.c
     up/pic.c)
 
 add_asm_files(lib_hal_xbox_asm ${HAL_XBOX_ASM_SOURCE})
 add_object_library(lib_hal_xbox ${HAL_XBOX_SOURCE} ${lib_hal_xbox_asm})
-if(NOT SARCH STREQUAL "xbox")
```

Comment:
Why did we have these lin es before? (cc @binarymaster )

Reply:
This was a hack ;)